Remove deprecated keyword HTTP/* for HTTP

nghiab1906724 opened the pull request.

jcamiel commented:
Hi @nghiab1906724 I don't see any rust code modified in this PR, do I miss something?

nghiab1906724 commented:
Hello @jcamiel ,
I apologize for this issue. I will add the missing code shortly.
Thank you so much!!!

nghiab1906724 commented:
Hi @jcamiel ,
I’ve just removed the deprecated HTTP/* keyword from the Rust code as requested.
Could you kindly review the changes?
Thank you for your support!

jcamiel commented on integration/hurl/tests_ok/deprecated.hurl:
```diff
@@ -1,4 +1,3 @@
-# HTTP/* is deprecated, use HTTP instead
 GET http://localhost:8000/hello
-HTTP/* 200
+HTTP 200
```
We can remove this integration test file `integration/hurl/tests_ok/deprecated.hurl` as there is no deprecation to test now

jcamiel commented on integration/hurlfmt/tests_export/http_version.hurl:
```diff
@@ -11,8 +11,4 @@ GET http://localhost:8000/hello
-# Deprecated
-GET http://localhost:8000/hello
-HTTP/* 200
+HTTP 200
```
This file must end with a new line

jcamiel commented on integration/hurlfmt/tests_export/http_version.lint.hurl:
```diff
@@ -11,8 +11,4 @@ GET http://localhost:8000/hello
-# Deprecated
-GET http://localhost:8000/hello
-HTTP/* 200
+HTTP 200
```
This file must end with a new line

jcamiel reviewed (changes requested):
Hi @nghiab1906724 

Thanks for the PR there are small changes to address. Could you please also rebase and squash the 2 commits to only one commit,

Thanks!

nghiab1906724 commented:
Hi @jcamiel  ,

I have made changes based on your comments. I’ve removed **"integration/hurl/tests_ok/deprecated.hurl"** and fixed the related issues.

Could you help me review it?  
If everything works well, I will rebase and squash the commits into a single commit.

Thanks!

jcamiel commented:
All tests are OK @nghiab1906724 you can rebase, squash and it should be good!

nghiab1906724 commented:
Hi @jcamiel ,
I’ve just rebased and squashed. Could you help me check?
Thank you so much!

jcamiel commented:
/accept

jcamiel commented:
/accept
